mllaunchpad/yaml_loader.py

```diff
@@ -0,0 +1,21 @@
+import yaml
+import os
+
```

Comment:
Nitpick: It somehow looks like this file has CRLF line endings instead of LF like the rest of the project. Could you change them to LF?